OloEngine/tests/Lua/LuaBindingTest.cpp

```
@@ -0,0 +1,1886 @@
+}
+
+// =============================================================================
+// Scene-backed integration tests (entity_utils with real Scene + Entity)
+// =============================================================================
+
+class ScopedTestSceneContext
+{
+  public:
+    explicit ScopedTestSceneContext(Scene* scene)
+    {
+        ScriptEngine::SetSceneContextForTesting(scene);
```

Clang (14.0.6)

[warning] variable 'scene' is not initialized

(cppcoreguidelines-init-variables)